packages/core/EventManager.ts

```diff
@@ -399,13 +400,21 @@ export default class EventManager {
         }
       }
     } else {
+      logger.silly(
+        "No destination Calendar found, falling back to first connected calendar",
+        JSON.stringify({
+          calendarCredentials: this.calendarCredentials,
+        })
+      );
+
       /**
        *  Not ideal but, if we don't find a destination calendar,
-       * fallback to the first connected calendar
+       *  fallback to the first connected calendar - Shouldn't be a CRM calendar
        */
-      const [credential] = this.calendarCredentials.filter((cred) => cred.type === "calendar");
+      const [credential] = this.calendarCredentials.filter((cred) => !cred.type.endsWith("other_calendar"));
```

Comment:
It was causing us to default to no calendar actually. cred.type can never be equal to 'calendar' within calendarCredentials as that is prepared by filtering credential having _calendar in type